packages/styles/scss/components/number-input/_number-input.scss

```diff
@@ -0,0 +1,479 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../colors' as *;
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/skeleton' as *;
+@use '../../utilities/button-reset';
```

Comment:
Just confirming this doesn't need the `as *`;

Reply:
nope!